redhawk/c/c_tree_converter.py

```diff
@@ -81,28 +82,28 @@ def UnescapeEscapeCharacters(self, tree):
   def ConvertFileast(self, tree):
     return N.Module(position = NP.NodePosition(self.filename, 1, 1),
         filename = self.filename,
-        children = map(self.ConvertTree, tree.children()))
+        children = list(map(self.ConvertTree, tree.children())))
```

Comment:
I would suggest adding a method to the base class (converter.tree_converter) called ConvertList which converts a list.

def ConvertList(nodes):
  return [self.ConvertTree(node) for node in nodes]

This way all these lines can be shortened to: children = self.ConvertList(tree.children())